pipelines/rj_segur__forca_municipal/utils.py

```
@@ -10,6 +12,23 @@
+def _add_id_hash(data: list[dict]) -> list[dict]:
+    """Adiciona id_hash a cada row com base nos dados brutos da API.
+
+    Hasheado antes da criação do DataFrame para evitar inconsistências de dtype
+    inference do pandas (ex: coluna inferida como int64 num run e float64 em outro
+    quando None aparece). json.dumps com sort_keys garante representação canônica
+    e determinística independente de tipo ou ordem de chaves.
+
+    Usa os 32 chars do MD5 (64 bits) — suficiente para o volume dessa API.
+    """
+    for row in data:
+        row["id_hash"] = hashlib.md5(
```

Ruff (0.15.13)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)